config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EncassEntityBuilder.java

```diff
@@ -32,6 +33,7 @@
 import static com.ca.apim.gateway.cagatewayconfig.util.properties.PropertyConstants.PALETTE_FOLDER;
 import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.*;
 import static java.lang.Boolean.FALSE;
+import static java.util.Collections.emptyList;
 import static org.apache.commons.lang3.ObjectUtils.firstNonNull;
 
 @Singleton
```

Comment:
I will add doc on the interface here, because all the implementations basically do the same.
Then I will add comment to refer javadoc in parent class.

Reply:
That sounds good great, on sub classes, can you please add something like, encass builder will build encass from this path and add to bundle? It helps create links how exported configurations are used.

Reply:
I prefer to go over explanations in all subclasses in a separate PR, @amoghsa.
If I do this here this PR will be a mix of many unrelated stuff, I think it's better to do in a separate PR.

Reply:
ok-sounds good.